Examples/ObjectiveC/InsetMapExample.m

```diff
@@ -0,0 +1,74 @@
+#import "InsetMapExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleInsetMap = @"InsetMapExample";
+
+@interface InsetMapExample () <MGLMapViewDelegate>
+@property (nonatomic) MGLMapView *mapView;
+@property (nonatomic) MGLMapView *miniMapView;
+@end
+
+@implementation InsetMapExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds];
+    self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    // Set the main map's center coordinate and zoom level.
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(18.1096,-77.2975)
+                zoomLevel:9
+                 animated:NO];
+
+    // Set inset mapview properties to create a smaller, non-interactive mapView that mimics the appearance of the main mapView.
```

Comment:
The length of the lines for comments should cap at 80 characters. The best way to visualize that limit is by enabling the page guide in Xcode's preferences:

<img height=300 src="https://user-images.githubusercontent.com/10850812/85344049-6f685980-b4a3-11ea-9bce-e78c9a9cd569.png">


We limit this length because we don't want users to have to scroll horizontally when these examples end up making it to the website. Please wrap these comments around to use the multi-line comment syntax if they go over this character limit:

```
/**
 This
 is
 a
 multi
 line
 comment :)
*/
```